app/controllers/form_controller.rb

```diff
@@ -0,0 +1,18 @@
+class FormController < ApplicationController
+  def index
+    @facilities=Facility.all
+  end
+
+  def create
+      fluff = (params[:type] =~ /item/) ? Item.new : Procedure.new
```

Comment:
change the name to something else more representative of what it is.

Reply:
For the type of form that you're trying to create here, it seems like using [polymorphic associations](https://guides.rubyonrails.org/association_basics.html#polymorphic-associations) could help.